VideoWeb/VideoWeb/ClientApp/src/app/services/connection-status.service.spec.ts

```diff
@@ -1,128 +1,241 @@
 import { fakeAsync, tick } from '@angular/core/testing';
-import { MockLogger } from '../testing/mocks/MockLogger';
+import { MockLoggerToConsole } from '../testing/mocks/MockLoggerToConsole';
 import { Logger } from './logging/logger-base';
 import { ConnectionStatusService } from '../services/connection-status.service';
+import { HttpClient, HttpErrorResponse, HttpResponse } from '@angular/common/http';
+import { TestBed } from '@angular/core/testing';
+import { of } from 'rxjs';
 
 describe('ConnectionStatusService', () => {
     let service: ConnectionStatusService;
-    const logger: Logger = new MockLogger();
+    let httpClientSpy: jasmine.SpyObj<HttpClient>;
+    const mockLoggerToConsole: Logger = new MockLoggerToConsole();
+    let TIME_TO_WAIT: number;
+
+    beforeEach(() => {
+        httpClientSpy = jasmine.createSpyObj('HttpClient', ['head']);
+
+        TestBed.configureTestingModule({
```

Comment:
i think @shaed-parkar has been trying to move us away from TestBed?

Reply:
Removed TestBed code